docs/application.md

```
@@ -2,49 +2,121 @@
 
 🇫🇷 [Lire en français](/mercator/fr/application)
 
-[![homepage.png](images/homepage.png)](images/homepage.png)
+*📑 Note: You can click on any image to increase its size*
+
+   [<img src="/mercator/images/homepage.png" width="700">](images/homepage.png)
+
+---
 
 ### Main page
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)